src/main/kotlin/eu/europa/ec/eudi/openid4vci/internal/http/CredentialEndpointClient.kt

```diff
@@ -47,11 +47,12 @@ internal class CredentialEndpointClient(
     suspend fun placeIssuanceRequest(
         accessToken: AccessToken,
         request: CredentialIssuanceRequest,
+        dpopNonce: String?
     ): Result<SubmissionOutcomeInternal> = runCatching {
         ktorHttpClientFactory().use { client ->
             val url = credentialEndpoint.value
             val response = client.post(url) {
-                bearerOrDPoPAuth(dPoPJwtFactory, url, Htm.POST, accessToken)
+                bearerOrDPoPAuth(dPoPJwtFactory, url, Htm.POST, accessToken, dpopNonce)
```

Comment:
shall we verify the dpopNonce is not null?

Reply:
We can do this on the client side. The library should work for the ones who dont need this, thats why it is non-nullable